commons-db/src/main/java/com/palantir/nexus/db/sql/BasicSQL.java

```diff
@@ -70,6 +71,7 @@
 import com.palantir.util.sql.VerboseSQLException;
 
 public abstract class BasicSQL {
+    private static final Logger sqlExceptionlog = Logger.getLogger("sqlException." + PreparedStatements.class.getName());
```

Comment:
Why not use the SqlLoggers.SQL_EXCEPTION_LOG?  The actual question being what is the difference between having it under PreparedStatements instead of BasicSql logger